Traceback (most recent call last):
  File "/home/torrent-user/flightmaster/src/./test.py", line 121, in <module>
    main()
  File "/home/torrent-user/flightmaster/src/./test.py", line 117, in main
    test_populate_profile(paths)
  File "/home/torrent-user/flightmaster/src/./test.py", line 50, in test_populate_profile
    profile_data = yaml.safe_load(f)
                   ^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/home/torrent-user/flightmaster/src/venv/lib/python3.11/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
    raise ParserError("while parsing a block mapping", self.marks[-1],
yaml.parser.ParserError: while parsing a block mapping
  in "profile.yml", line 1, column 1
expected <block end>, but found '<block mapping start>'
  in "profile.yml", line 2, column 3
